### DISCUSSION TITLE
Add graph option to show timetables

### DISCUSSION THREAD

**Discussion by pablo03v** [patrykandpatrick/vico#561] [Ideas]
I'm coding an app that logs timeframes. Say a user works a few hours every day. I can show a graph on how much they worked each day but not when they started working and ended working.

Existing chart:
<img width="518" alt="image" src="https://github.com/patrykandpatrick/vico/assets/93644977/d2861fac-8395-4fcd-a153-cd5a589c6d0f">

My wish:
![Untitled 3](https://github.com/patrykandpatrick/vico/assets/93644977/073eb82d-ab9f-4fb2-b323-284adba9b9d3)

**Comment by patrickmichalik** [comment]
Hello! This is a planned addition. Please see #430 and the update [here](https://github.com/patrykandpatrick/vico/discussions/430#discussioncomment-8337990).

**Comment by patrickmichalik** [comment]
Please see [here](https://github.com/patrykandpatrick/vico/discussions/430#discussioncomment-8360061) for a brief overview of a possible workaround. If you have any questions, feel free to let me know.
